Makefile

```diff
@@ -435,7 +435,11 @@ test: lint karma
 
 .PHONY: runserver
 runserver: remove_old_config
-	$(REPO_BIN_DIR)/run-server.sh
+	@if [[ -z "${NO_WATCH}" ]]; then \
+	  $(REPO_BIN_DIR)/run-server.sh; \
+	else \
+	  node bin/server.js; \
+	fi
```

Comment:
Might be cleaner to just have a separate make target, e.g., `runserver-only` or `runserver-nowatch` instead of environment variables.